### PR TITLE
community: fix qianfan chat stream calling caused exception

### PR DESCRIPTION
**Pull request by danielhjz**
- **Description:** 
`QianfanChatEndpoint`  extends `BaseChatModel` as a super class, which has a default stream implement might concat the MessageChunk with `__add__`. When call stream(), a ValueError for duplicated key will be raise.
  - **Issues:** 
     * #13546  
     * #13548
     * merge two single test file related to qianfan.
  - **Dependencies:** no
  - **Tag maintainer:**

### CODE REVIEW

**Comment by danielhjz** [comment]
@efriis PTAL fix issue about qianfan ChatModel

File: libs/langchain/langchain/chat_models/baidu_qianfan_endpoint.py [hunk 1]
```diff
@@ -97,14 +97,14 @@ class QianfanChatEndpoint(BaseChatModel):
...
-    penalty_score: Optional[float] = 1
+    top_p: Optional[float] = None
+    temperature: Optional[float] = None
+    penalty_score: Optional[float] = None
     """Model params, only supported in ERNIE-Bot and ERNIE-Bot-turbo.
     In the case of other model, passing these params will not affect the result.
     """
 
-    model: str = "ERNIE-Bot-turbo"
+    model: Optional[str] = None
```
[the remark this answers is not in the document]
Comment (danielhjz) [reply libs/langchain/langchain/chat_models/baidu_qianfan_endpoint.py:107]: In Fact, API will take the same default values of (top_p, temperature, penalty_score) to these. And SDK will take a deefault model with ERNIE-Bot-turbo, it should not be a breaking change. But that might be unrelated to this fix. I will remove these apart from the content of fix, and create another PR for this. best regards.

**Comment by danielhjz** [comment]
@efriis  update committed, PTAL

**Comment by efriis** [comment]
Hey @danielhjz ! Just a heads up this won't go in until next week due to some ongoing refactor work on these folders. Will follow up then, and may require some updating of paths!

**Comment by danielhjz** [comment]
> Hey @danielhjz ! Just a heads up this won't go in until next week due to some ongoing refactor work on these folders. Will follow up then, and may require some updating of paths!

Thanks for message @efriis. Waiting for the next updating, and i may be able to help updating if needed.

**Comment by efriis** [comment]
It's in, and looks like some paths need updating. let me know when that's in!

**Comment by danielhjz** [comment]
Updating job done, and i also merge the integrations_tests into one file. PTAL @efriis

**Comment by danielhjz** [comment]
Sry for delay of updating, CI checks have been accomplished.  Leave comments if you have any opinion about this PR. best regards. @efriis  cc @baskaryan

**Comment by danielhjz** [comment]
@efriis Comment me if there is anything i could promote the merging of this PR.

**Comment by danielhjz** [comment]
Reset the issue to template <package>:<description>.

**Comment by danielhjz** [comment]
@efriis Hi, Ive noticed this PR is labeled with `template`. Tell me if anything wrong.

**Comment by danielhjz** [comment]
@efriis @hwchase17  😭 Shall we give some feedbacks.

**Review by efriis — APPROVED** [review]

**Pull request merged**